core/src/main/java/greencity/config/SecurityConfig.java

```diff
@@ -153,6 +153,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 EVENTS,
                 EVENTS + "/event/{eventId}",
                 EVENTS + "/getAllSubscribers/{eventId}",
+                EVENTS + "/getAllFavoriteEvents",
```

Comment:
how can unauthorized users have favorite events?

Reply:
we should add the `userId` to the request or return the favorite events for authorized user only